src/store/d-apps/d-apps-epics.ts

```diff
@@ -142,6 +151,36 @@ const approveOperationRequestEpic = (action$: Observable<Action>) =>
     )
   );
 
+const waitForOperationCompletionEpic = (action$: Observable<Action>) =>
+  action$.pipe(
+    ofType(waitForOperationCompletionAction),
+    toPayload(),
+    withLatestFrom(tezos$),
+    switchMap(([{ opHash, sender }, tezos]) =>
+      from(tezos.operation.createOperation(opHash)).pipe(
+        switchMap(operation => from(operation.confirmation(1))),
+        switchMap(({ completed }) =>
+          completed
+            ? of(null).pipe(
+                delay(15000),
+                concatMap(() => [
+                  loadTezosBalanceActions.submit(sender),
+                  loadTokenBalancesActions.submit(sender),
+                  loadActivityGroupsActions.submit(sender),
+                  loadSelectedBakerActions.submit(sender)
+                ])
+              )
+            : throwError({ message: "Transaction wasn't completed" })
```

Comment:
have you tested this case? does it shows error toast?

Reply:
It looks like it should never happen (see https://github.com/ecadlabs/taquito/blob/77bf9da3bc4b72e9afce51c815f5ccd2a2a5580f/packages/taquito/src/wallet/operation.ts#L138)
I'll remove it